src/app/app-routing.module.ts

```diff
@@ -22,6 +19,7 @@ const routes: Routes = [
   { path: 'signup', loadChildren: './pages/signup/signup.module#SignupPageModule' },
 
 
+  { path: 'home', loadChildren: './pages/home/home.module#HomePageModule' },
```

Comment:
This way, the user can be sent to "home" and the home routing module can decide which specific tab that means. Currently, it's set to go to the cards page but the user may be able to set it to any other via the settings page.